sdk/src/main/java/com/horizen/cryptolibprovider/ThresholdSignatureCircuitImplZendoo.java

```diff
@@ -100,7 +95,7 @@ public byte[] generateMessageToBeSigned(List<WithdrawalRequestBox> bt,
     }
 
     @Override
-    public Pair<byte[], Long> createProof(List<WithdrawalRequestBox> bt,
+    public Pair<byte[], Long> createProof(List<BackwardTransfer> backwardTransfers,
```

Comment:
We can remove the TODO comment. Btw, the `zk` boolean parameter is true if the proof is a Zero Knowledge proof